Document two new feature gates around CEL cost enforcement (#47321)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates/strict-cos-enforcement-for-webhooks.md

---
title: StrictCostEnforcementForWebhooks
content_type: feature_gate
_build:
  list: never
  render: false

stages:
  - stage: alpha
    defaultValue: false
    fromVersion: "1.31"
    
---

> Enable the enforcement of the strict cost calculation for CEL extended libraries in `matchConditions` field under Webhooks.

Reviewer: Even better
```suggestion
Enable enforcement of the strict cost calculation for CEL extended libraries within the
`matchConditions` field for
[admission webhooks](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks).
```

Author: Thanks! Updated :)

---
title: StrictCostEnforcementForWebhooks
content_type: feature_gate
_build:
  list: never
  render: false

stages:

>   - stage: alpha

Reviewer 2: Similarly, whether this `StrictCostEnforcementForWebhooks` feature gate is in the alpha or beta stage?

	genericfeatures.StrictCostEnforcementForWebhooks: {Default: false, PreRelease: featuregate.Beta},
	
[Referring to kubernetes/kubernetes/pkg/features/kube_features.go#L1302 (here)](https://github.com/kubernetes/kubernetes/blob/6fc0a69044f1ac4c13841ec4391224a2df241460/pkg/features/kube_features.go#L1302)

    defaultValue: false

>     fromVersion: "1.30"

Reviewer: ```suggestion
    fromVersion: "1.31"
```
?
